### PR TITLE
Add condition to protect IndexedLineSet when no coord

### PR DESCRIPTION
**Pull request by remi-cyberbotics**
This pull-request fixes issue #1443

### CODE REVIEW

File: src/webots/nodes/WbIndexedLineSet.cpp [hunk 1]
```diff
@@ -144,6 +145,8 @@ void WbIndexedLineSet::reset() {
 
 int WbIndexedLineSet::computeCoordsData(float *data) {
   WbMFInt::Iterator it(*mCoordIndex);
+  if (it.hasNext() == 0)
```
Comment (DavidMansolino) [inline src/webots/nodes/WbIndexedLineSet.cpp]:
```suggestion
  if (!it.hasNext())
```

This looks simpler / easier to understand to me.

Comment (remi-cyberbotics) [reply src/webots/nodes/WbIndexedLineSet.cpp]: Definitely, thanks for your update

File: docs/reference/changelog-r2020.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ Released on XXX.
     - Fixed Matlab API.
     - Fixed missing stdout/stderr flush when a controller is changed or restarted while simulation is running (thanks to tsampazk).
     - Fixed the ROS controller of the Universal Robots UR3e, UR5e and UR10e to send the success state when a trajectory succeeded (thanks to Tim-Stoll).
+    - Fixed a crash when trying to add IndexedLineSet to a geometry's shape node with no coordinates.
```
Comment (DavidMansolino) [inline docs/reference/changelog-r2020.md]:
```suggestion
    - Fixed a crash when trying to add an [IndexedLineSet](indexedlineset.md) node to a geometry's [Shape](shape.md) node with no coordinates.
```

Sorry for being picky.

Comment (DavidMansolino) [reply docs/reference/changelog-r2020.md]: I am also not sure about, `coordinates`, should we remove the `s`?

Comment (omichel) [reply docs/reference/changelog-r2020.md]: I would leave it plural, see [this](http://www.englishlessonsbrighton.co.uk/followed-singular-plural-noun/).

**Pull request merged**